themes/scss/widgets/canvas.scss

```diff
@@ -60,18 +60,27 @@
 
 // Widget media query
 @include media-query() {
+
+.CDB-Dashboard-mapWrapper {
+  flex: 1;
+}
+  .CDB-Dashboard-canvas {
+    position: absolute;
+  }
+
   .CDB-Widget-canvas {
     @include display-flex();
     @include flex-wrap(nowrap);
-    width: calc(100% - 80px);
-    min-width: calc(100% - 80px);
-    max-width: calc(100% - 80px);
-    height: 350px;
+    width: calc(100% - 88px);
```

Comment:
Properties should be ordered flex, width, min-width, max-width, height, margin-top, margin-left, border-radius